client/src/services/helper.js

```diff
@@ -16,11 +16,27 @@ export const makeRequest = (path, options) => {
     if (!options.headers) {
       options.headers = {};
     }
-    options.headers['Authorization'] =  `Bearer ${ token }`;
+    options.headers['Authorization'] = `Bearer ${token}`;
   }
 
-  return fetch(BASE_URL + path, options).then((response) => {
+  return fetch(BASE_URL + path, options).then(response => {
     return response.json();
   });
-
-};
\ No newline at end of file
+};
+
+export const makeRequest = async (path, options) => {
+  const response = await makeFetchRequest(path, options);
+  if (response.code && response.code == 'token_not_valid') {
```

Comment:
id check status 401

Reply:
for that we will have to tweek makeFetchRequest function, that will break many listening responses.

Reply:
you can wrap around the response of makeFetchRequest function, nothing will break